UM/Preferences.py

```diff
@@ -34,11 +34,17 @@ def __init__(self) -> None:
 
         self._parser = None  # type: Optional[configparser.ConfigParser]
         self._preferences = {}  # type: Dict[str, Dict[str, _Preference]]
-        self._untrusted_settings: Dict[(str, str), Callable] = {}
+        self._untrusted_preferences: Dict[(str, str), Callable] = {}
 
-    def indicateUntrustedSetting(self, group: str, key: str, eval_func: Callable) -> None:
+    def indicateUntrustedPreference(self, group: str, key: str, eval_func: Callable) -> None:
```

Comment:
```suggestion
    def indicateUntrustedPreference(self, group: str, key: str, eval_func: Callable[[str], bool]) -> None:
```
Although it could be that this trips up if you put a lambda in it. I'm not entirely sure of that.

Reply:
Well it runs, and is furthermore a valid way of typing lambda's (at least after you've put them in a variable, the lambda can't be typed that way during declaration) according to this stackoverflow answer: https://stackoverflow.com/a/33833896